edgelet/build/linux/package.sh

```diff
@@ -33,6 +33,24 @@ case "$PACKAGE_OS" in
         esac
         ;;
 
+    'redhat8')
+        # Converts debian versioning to rpm version
+        # deb 1.0.1~dev100 ~> rpm 1.0.1-0.1.dev100
+        RPM_VERSION="$(echo "$VERSION" | cut -d"~" -f1)"
+        RPM_TAG="$(echo "$VERSION" | cut -s -d"~" -f2)"
+        if [ -n "$RPM_TAG" ]; then
+            RPM_RELEASE="0.$REVISION.$RPM_TAG"
+        else
+            RPM_RELEASE="$REVISION"
+        fi
+
+        case "$PACKAGE_ARCH" in
+            'amd64')
+                DOCKER_IMAGE='redhat/ubi8:latest'
```

Comment:
Does the `latest` mean the latest stable?

Reply:
I think so - This will be the latest RedHat 8 release, currently latest version is 8.5.